packages/admin-panel/src/dataTables/components/PreviewFilters/filters/BooleanField.js

```diff
@@ -9,20 +9,20 @@ import React from 'react';
 import { Select } from '@tupaia/ui-components';
 import { ParameterType } from '../../editing';
 
-export const BooleanField = ({ name, value, onChange }) => {
-  if (!value) {
-    onChange(false);
-  }
+export const BooleanField = ({ name, value, onChange, config }) => {
+  const defaultValue = config?.hasDefaultValue ? config?.defaultValue.toString() : null;
```

Comment:
I got an error here "Uncaught TypeError: Cannot read properties of undefined (reading 'toString')"